src/mixins/input.js

```diff
@@ -194,11 +205,18 @@ export default {
         wrapperChildren.push(this.genIcon('append', defaultAppendCallback))
       }
 
-      if (this.asyncLoading) {
+      if (this.asyncLoading !== false) {
         detailsChildren.push(this.$createElement('v-progress-linear', {
           props: {
-            indeterminate: true,
-            height: 2
+            indeterminate: this.asyncLoading === true,
+            value: Number(this.asyncLoading),
+            height: this.asyncLoadingHeight,
+            primary: this.asyncLoadingColor === 'primary',
```

Comment:
Since we are trying to move away from this model, I'd be interested to see if we could modify the color differently.